### DISCUSSION TITLE
Your ideas on having hidden checkboxes to add functionality?

### DISCUSSION THREAD

**Discussion by Yohn** [Yohn/PicoCSS#19] [Polls]
Originally I was against having hidden checkbox to do certain functionality on a website, Mainly because I felt we could just use javascript for those features if need be.

I understand the need to not use JavaScript with this project, although I may add some features that would require javascript, I would still make those features optional, and not be mandatory to use the javascript provided.

With the hamburger nav, to do that without javascript we had to use a hidden checkbox to achieve the results. 
Since we just introduced this hidden checkbox option now, I was wondering what everyone's thoughts are about using them?

**Comment by odama626** [comment]
I don't like the idea of hidden elements or js, I think it should be approached with attributes like dialogs are

a class of open, or an aria attribute or the like
